librz/core/cmd_analysis.c

```diff
@@ -780,6 +780,38 @@ static bool listOpDescriptions(void *_core, const char *k, const char *v) {
 	return true;
 }
 
+RZ_API RzOutputMode setMode(char input) {
```

Comment:
I think this should be moved to `cmd_api.c` similarly to `rz_output_mode_to_char` and you can use a similar logic to transform from char to mode. You can also add it to `core_private.h` so others will be able to use it as well.

Reply:
I suggest the `rz_output_mode_of_char()` name then.